### ISSUE TITLE
Navigation on Garmin Watch

### ISSUE THREAD

**Issue by adamcarter81** [organicmaps/organicmaps#7382]
I tried to use OM when walking recently (around London) to replace other map solutions (on Android). It would be really useful if I could leave the phone in my pocket and just get the directions notifications on my watch (Garmin Fenix 6), to save me having to be holding my phone.

Although OM did try to send some notification to the watch, these didn't seem helpful - with maybe just a road name, and often the notifications didnt' even arrive. This was using the default OM notifications in Android and just having OM selected as a notificaiton app in Garmin Connect.  

This works well with say for example Google Maps, and then a companion app on the phone to format the notifications and pass them to the Garmin app on the phone to show them. Downside to this is that it requires two extra apps (or at least an extra service in OM, and the Garmin app).

Another alternative is Locus Maps - they achieve it similarly with a Garmin app and a plugin inside Locus. This works quite well also.

One solution could be to pass just the destination to Garmin Explore (I tried to share it but Garmin Explore didn't show up), or to export the track as (as a GPX?) and then to Garmin Explore - this would get the route or destination on the watch quickly.

Another could be to just use the existing notifications - but somehow get them more reliable, faster updates and to give more information on the next turn etc.

TLDR - I'm looking to use OM to navigate when in my pocket and have notifications sent to my Garmin watch
